packages/near-membrane-dom/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import './custom-devtools-formatter';
```

Comment:
can you move this functionality to the near-membrane-base? I want to use it in my downstream package

Reply:
@Jack-Works near-membrane-base is intended to be host agnostic, so moving browser devtools related code into that package might not make sense. I wonder if we could/should just move this into `near-membrane-shared`—WDYT @jdalton?

Reply:
This is really for browser environments (Chromium based) which is why it lives in `near-membrane-dom`. This could be used as a reference though.

Reply:
Yeah, I know it's for the Chromium. My [downstream package](https://github.com/DimensionDev/aot-secure-ecmascript/tree/main/packages/membrane/src) is also running on Chromium (but I cannot use `near-membrane-dom` because I cannot use iframe). I want to reuse the dev tools formatted code without implementing it myself or copying it.

Reply:
that will be great!